puppet/lib/puppet/indirector/catalog/puppetdb.rb

```diff
@@ -48,6 +49,19 @@ def redact_sensitive_params(hash)
     nil
   end
 
+  def munge_catalog_inputs(hash)
+    inputs = hash.delete('inputs')
+
+    return if inputs.nil?
+
+    {
+      certname: hash['certname'],
+      catalog_uuid: hash['catalog_uuid'],
+      producer_timestamp: hash['producer_timestamp'],
+      inputs: inputs
+    }
+  end
```

Comment:
@pcarlisle this is the proposal for how the inputs are getting to PDB, via the catalog indirection. Thoughts?